pages/osx/wc.md

```diff
@@ -5,19 +5,19 @@
 
 - Count lines in file:
 
-`wc -l {{path/to/file}}`
+`wc --lines {{path/to/file}}`
```

Comment:
Revert this file. macOS doesn't support long arguments
```suggestion
`wc -l {{path/to/file}}`
```
Maybe we should use this link: https://ss64.com/osx/wc.html

Reply:
Fixed ;)